_data/projects.yml

```diff
@@ -49,6 +49,10 @@
       spoken_languages:
         - EN
         - DE
+    - name: Simon Schrottner
```

Comment:
You can also add a social media URL if you want
(see [example](https://github.com/hack-commit-push/paris2021/blob/708a6576b775f8ee91866859ef80a3878c31fd71/_data/projects.yml#L80))

Reply:
thank you, added